File: 프로그래머스/weeklyChallenge/001.cpp
#include <iostream>

using namespace std;

long long solution(int price, int money, int count)
{
    int answer = 0;
    int sum = 0;
    for(int i=1;i<=count;i++){
        sum = sum + i*price;
    }
    if(money < sum){
        answer = sum - money;
    }
    return answer;
}

File: 프로그래머스/weeklyChallenge/001.cs
using System;

class Solution
{
    public long solution(int price, int money, int count)
    {
        long sum = 0;
        long ret = 0;
        //Console.WriteLine(price);
        for(int i=1;i<=count;i++){
            sum = sum + i * price;
            Console.WriteLine("sum : "+sum+", price : "+i*price);
            
        }
        Console.WriteLine(sum);
        if(money < sum){
            ret = sum - money;
        }
        return ret;
    }
}